app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,70 @@
+class OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[show edit update destroy]
+  before_action :set_cities, only: %i[new create edit update]
+
+  def index
+    @organizations = Organization.all
+  end
+
+  def show
+  end
+
+  def new
+    @organization = Organization.new
+  end
+
+  def edit
+  end
+
+  def create
+    @organization = Organization.new(organization_params)
+    saved = service.save_organization_and_cities_organization(organization: @organization,
+                                                               city_ids: city_ids)
+
+    redirect_to organization_path(@organization), notice: 'Organization was successfully created.' if saved
+    render :new, status: :unprocessable_entity unless saved
+  end
+
+  def update
+    respond_to do |format|
+      if @organization.update(organization_params)
+        format.html { redirect_to @organization, notice: "Organization was successfully updated." }
+        format.json { render :show, status: :ok, location: @organization }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @organization.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def destroy
+    @organization.destroy
```

Comment:
Destory can fail and return false you know - https://devdocs.io/rails~6.1/activerecord/persistence#method-i-destroy